cloudify-manager-worker/CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ Provides a release information for each version splitted by the next sections
 - Helm charts **bitnami/rabbitmq** and **bitnami/postgresql** now dependent charts and may be deployed automatically, so isn't necessary to deploy them manually. For new installation we suggest to add parameters **rabbitmq.deploy: true** and **postgresql.deploy: true** for enable deploy rabbitmq and postgresql as dependencies. We left defaults for these parameters to **false** for backward compatibility (will be changed later).
 - "cloudify-manager-worker/README.md" file updated.
 - PostgreSQL will create PV by default (when postgres deploy enabled in parameter **postgresql.deploy: true**).
+- RabbitMQ will create PV by default (when postgres deploy enabled in parameter **rabbitmq.deploy: true**).
```

Comment:
typo, should be "...(when **rabbitmq** deploy enabled ..."

Reply:
Oh sure, fixed